### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by lboeman**
Adds a brief intro to the project and link to dev documentation.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,30 @@
...
+Solar Performance Insight is an open source platform for evaluating solar power
+plant performance. Solar Performance Insight aims to provide an alternative
+to available asset management tools for PV Operations and Mainteneance (O&M)
+providers where existing tools are unaffordable or impractical.
+
+## Platform Layout
+
+The Solar Performance Insight platform has two main components:
+
+- *REST API*
```
Comment (wholmgren) [inline README.md:12]:
```suggestion
### REST API
```

File: README.md [hunk 2]
```diff
@@ -1 +1,30 @@
...
+  Documentation for the development instance of the API can be found at
+  [dev.solarperformanceinsight.org/api/docs](https://dev.solarperformanceinsight.org/api/docs).
+  Source code can be found in the [api](tree/main/api) directory.
+
+  The API utilizes a MySQL database for storage and uses the [dbmate](https://github.com/amacneil/dbmate)
+  migration utility for keeping the database in sync during development.
+  Files related to the database and *dbmate* migrations can be found in the
+  [db](tree/main/db) directory.
+
+- *Web Dashboard*
```
Comment (wholmgren) [inline README.md:25]:
```suggestion
### Web Dashboard
```

File: README.md [hunk 3]
```diff
@@ -1 +1,30 @@
...
+The Solar Performance Insight platform has two main components:
+
+- *REST API*
+
+  A REST API to enable programmatic access to Solar Performance Insight
+  services that is built using [FastAPI](https://fastapi.tiangolo.com/).
+  Documentation for the development instance of the API can be found at
+  [dev.solarperformanceinsight.org/api/docs](https://dev.solarperformanceinsight.org/api/docs).
+  Source code can be found in the [api](tree/main/api) directory.
+
```
Comment (wholmgren) [inline README.md:19]: something about python library that does the work

File: README.md [hunk 4]
```diff
@@ -1 +1,30 @@
...
+  The API utilizes a MySQL database for storage and uses the [dbmate](https://github.com/amacneil/dbmate)
```
Comment (wholmgren) [inline README.md:20]:
```suggestion
  The API stores metadata and job results in a MySQL database. It uses the [dbmate](https://github.com/amacneil/dbmate)
```

File: README.md [hunk 5]
```diff
@@ -1 +1,30 @@
...
+  migration utility for keeping the database in sync during development.
+  Files related to the database and *dbmate* migrations can be found in the
+  [db](tree/main/db) directory.
+
+- *Web Dashboard*
+
+  Web front end provides a graphical interface to the Solar Performance
```
Comment (wholmgren) [inline README.md:27]:
```suggestion
  The web front end provides a graphical interface to the Solar Performance
```

File: README.md [hunk 6]
```diff
@@ -1 +1,30 @@
...
+  Insight services build using [Vue.js](https://vuejs.org/).The development
```
Comment (wholmgren) [inline README.md:28]:
```suggestion
  Insight services. The front end is built using [Vue.js](https://vuejs.org/). The development
```

**Review by wholmgren — COMMENTED** [review]
Thanks, minor stuff below...

File: README.md [hunk 7]
```diff
@@ -1 +1,36 @@
...
+
+## Platform Layout
+
+The Solar Performance Insight platform has two main components:
+
+### REST API
+
+  A REST API to enable programmatic access to Solar Performance Insight
+  services that is built using the [FastAPI](https://fastapi.tiangolo.com/)
+  Python web framework. The [pvLib python](https://pvlib-python.readthedocs.io/)
```
Comment (wholmgren) [inline README.md:16]:
```suggestion
  The REST API enables programmatic access to Solar Performance Insight
  services. It is built using the [FastAPI](https://fastapi.tiangolo.com/)
  Python web framework. The [pvlib python](https://pvlib-python.readthedocs.io/)
```

**Review by wholmgren — APPROVED** [review]
@cwhanse do you want us to add anything else to the README?

File: README.md [hunk 8]
```diff
@@ -1 +1,36 @@
-# solarperformanceinsight
+# Solar Performance Insight
+
+Solar Performance Insight is an open source platform for evaluating solar power
+plant performance. Solar Performance Insight aims to provide an alternative
+to available asset management tools for PV Operations and Mainteneance (O&M)
+providers where existing tools are unaffordable or impractical.
```
Comment (cwhanse) [inline README.md:6]:
```suggestion
to available asset modeling tools for PV Operations and Mainteneance (O&M)
providers where existing tools are unaffordable or impractical.
```

Asset management is probably understood to encompass maintenance scheduling, parts ordering, billing, etc. all aspects of keeping a PV system operational and generating revenue. We only have modeling and analysis tasks in our scope.

**Review by cwhanse — APPROVED** [review]

**Pull request merged**